EIPS/eip-XXX.md

```diff
@@ -0,0 +1,415 @@
+---
+eip: TODO
+title: Ethereum state using a unified binary tree
+author: Vitalik Buterin (@vbuterin), Guillaume Ballet (@gballet), Dankrad Feist (@dankrad), Ignacio Hagopian (@jsign), Kevaundray Wedderburn (@kevaundray), Tanishq Jasoria (@tanishqjasoria), Gajinder Singh (@g11tech), Danno Ferrin (@shemnon), Piper Merriam (@pipermerriam), Gottfried Herold (@GottfriedHerold)
+discussions-to: TODO
+status: Draft
+type: Standards Track
+category: Core
+created: 2025-01-20
+
+---
+
+## Abstract
+
+Introduce a new binary state tree, intended to replace the hexary patricia trees. Account and storage tries are merged into a single tree with 32-byte keys, which also contains contracts code. Account data is broken into independent leaves which  are grouped by 256 in order to provide some locality.
+
+(Draft note: the hash function used in the current draft is not final, other potential candidates are Keccak or Poseidon2. For the latter, an extra section on how to encode 32-bytes->[FiniteField] will be created. **Do not** assume Blake3 is a final decision.)
```

Comment:
This is the only "draft" note in the EIP that will eventually be removed.
I wanted to signal that the hash function might change in further drafts so potential PoC have that in mind.